ajax/libs/coolprop/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "coolprop",
+  "description": "CoolProp is a thermophysical property database. It offers similar functionality to REFPROP",
+  "version": "6.3.0",
```

Comment:
Please remove the version attribute, the auto-update bot will populate this itself when it pulls in versioned files